### PR TITLE
Fix/options

### PR DESCRIPTION
**Pull request by jgfidelis**
Style fixes (for inline mode only) for this:
screen shot 2019-01-28 at 8 15 28 pm

Workspace para teste: https://testemini--delivery.myvtex.com/

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/AttachmentList.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ class AttachmentList extends Component {
   formatAttachmentName = (quantity, name) =>
     this.props.intl.formatMessage({ id: 'editor.productSummary.attachmentName' }, { quantity, name })
 
-  // TODO:  not sure if it is the best way to define if an item should be displayed
-  canShow = (option) => option.optionType !== 'Basic Toppings'
+  // Not showing free stuff you get as a basic assembly option
+  canShow = option => getProductPrice(option) > 0 || option.optionType === 'Crust'
```
Comment (jgfidelis) [inline react/components/AttachmentList.js]: é a únca maneira de resolver o edge case onde a "Classic Crust" vem com preço 0

File: react/index.js [hunk 2]
```diff
@@ -227,13 +227,11 @@ class ProductSummary extends Component {
...
       'flex items-start',
       {
-        'justify-center': displayMode !== 'inline',
-        'justify-left w-100': displayMode === 'inline',
+        [`${productSummary.nameContainer} justify-center`]: displayMode !== 'inline',
+        'justify-left w-100 pt5': displayMode === 'inline',
         'pv5': displayMode === 'small',
         't-mini pb2': displayMode !== 'normal',
         'pv6': displayMode === 'normal',
-        [productSummary.nameContainer]: !showBorders,
```
Comment (jgfidelis) [inline react/index.js]: corrigindo estilos que prs passadas quebraram durante rebases errados

File: react/components/AttachmentList.js (hunk 1 above)
Comment (lbebber) [reply react/components/AttachmentList.js]: marca um TODO aqui, esse crust nao pode ficar aqui a longo prazo

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- fix styles for inline mode broken by previous wrong rebases
```
Comment (luciannojunior) [inline CHANGELOG.md]: Capitalize `fix`

**Pull request merged**